content/kit/webcam.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Webcam
+---
+
+To see the outside world, a webcam is used to capture images for them to be processed. The webcam in the kit is a [TeckNet C016](http://www.tecknet.co.uk/c016.html).
+
+![The Webcam](http://www.tecknet.co.uk/media/catalog/product/cache/1/thumbnail/9df78eab33525d08d6e5fb8d27136e95/c/0/c016_webcam_4_.jpg?width=30pc)
+
+
+## Focus
+The webcam has a manual focus ring around the lens. During standard operation, this lens may be knocked, and move the image out of focus, which can seriously reduce the accuracy of the [vision](/api/vision/). 
+
+It's recommended to manually focus the camera using a separate computer, and then use a __non permanent__ method (hot glue is great for this)  
+
+{{% notice warning %}}
+There's no guarantee the webcam is correctly focused in your kit!
+{{% /notice %}}
+
+## Mounting the camera
+The camera can be mounted in many different ways, from cable ties to duct tape. We recommend using the 1/4" screw mount on the base of the camera, commonly used for tripods. This has the advantage of being far sturdier than other methods and less prone to moving, whilst reducing any risk of damaging the camera. 
+
+## The LEDs
+The front of the webcam has 6 LEDs. These LEDs are fairly weak, however may increase reliability of image processing in close, dimly lit environments.
+
+The LEDs are controlled using a dial inline on the USB cable. Although it's a dial, the LEDs only have an _on_ and _off_ state, no granular control. This also means it's not possible to control the LEDs from the [vision API](/api/vision). 
```

Comment:
This could be read that the lack of API control is related to the LEDs only being on/off, rather than the dial.

Maybe drop the last sentence and make the first sentence something like: "The LEDs are controlled only using the dial inline on the USB cable."